src/platform/packages/shared/response-ops/alerts-fields-browser/components/field_items/field_items.tsx

```diff
@@ -67,28 +67,30 @@ export const getFieldItemsData = ({
     selectedCategoryIds.length > 0 ? selectedCategoryIds : Object.keys(browserFields);
   const selectedFieldIds = new Set(columnIds);
 
-  const fieldItems = uniqBy(
-    'name',
-    categoryIds.reduce<BrowserFieldItem[]>((fieldItemsAcc, categoryId) => {
+  const getFieldItems = () => {
+    let fieldItemsAcc: BrowserFieldItem[] = [];
+    for (let i = 0; i < categoryIds.length; i += 1) {
+      const categoryId = categoryIds[i];
       const categoryBrowserFields = Object.values(browserFields[categoryId]?.fields ?? {});
       if (categoryBrowserFields.length > 0) {
-        fieldItemsAcc.push(
```

Comment:
At scale this was crashing the browser:
<img width="522" alt="Screenshot 2025-02-22 at 12 47 53 AM" src="https://github.com/user-attachments/assets/81e51044-58e3-4f05-bc65-5ed5f98eb860" />

Reply:
<img width="461" alt="Screenshot 2025-02-22 at 12 47 59 AM" src="https://github.com/user-attachments/assets/9527f5b6-9622-4b37-9930-c58ff7f9e6da" />